react-native-reanimated.d.ts

```diff
@@ -463,20 +465,19 @@ declare module 'react-native-reanimated' {
     export function scrollTo(ref: RefObject<ReactNativeScrollView | ScrollView>, x: number, y: number, animated: boolean): void;
 
     // gesture-handler
-    type OnGestureEvent = (event: PanGestureHandlerGestureEvent) => void;
+    type OnGestureEvent<T> = (event: T) => void;
```

Comment:
I wonder if it shouldn't be `T extends GestureHandlerGestureEvent` to narrow type down. Writing this here, but applies to changes in `Handler` and `GestureHandlers` types.

Reply:
Yep, it can be, it won't change anything in the current implementation, but may help in the future.